### DISCUSSION TITLE
Tft display not working on stm32h723

### DISCUSSION THREAD

**Discussion by mas3058** [stm32duino/Arduino_Core_STM32#2284] [Q&A] [answered]
I have an TFT display running blazingly on a teensy 4.1 It uses the Adafruit library with a simple demo program running in the Arduino IDE.  I took the same display and moved it to the stm32h723.  Setup the drivers and the display will not function.  Scoped the signals    SPI1,  SCLK flying, MOSI flying, RESET HIGH, MISO not necessary, Data Command Flying,  CS dead,  should be flying,  Wrote a sketch to toggle all those GPIOs they all toggle fine.

Scratching my head,  If anyone has any Idea Please let me know

**Comment by fpistm** [comment]
hI @mas3058 
I've tested with Nucleo H723ZG and an adafruit graphic display and see no issue using this example: https://github.com/adafruit/Adafruit-ST7735-Library/tree/master/examples/graphicstest
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/20641798/2e9003c3-552e-4d55-a5f7-7d99ad9be2ad)
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/20641798/8b79f2d2-de94-42b5-8226-194291a6a683)
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/20641798/f0d178e1-8443-4d58-90c1-c2463160e860)

The only restriction I know is this one:
https://github.com/adafruit/Adafruit-GFX-Library/pull/359#issuecomment-1886367743

**Comment by mas3058** [comment, marked as answer]
Just an update, I finally got the display working, a few rookie problems that screwed me up for two weeks.  The board I am using is a cheap but nicely made board from China.  The STM32H723 board was 6 dollars, so I bought a few of them.  The first issue was no schematic which was painful.  I did eventually receive from the manufacturer of the board.   I could not get (Chip Select) to work off of PA4, I did get it to work at an alternate pin PG10.  In the process of writing the test sketch I eliminated the SPI.begin() and tft.begin() functions by mistake.  Then I had a few problems with the TFT_eSPI.h library but its all good now.  Here is the schematic and some note that may help someone.


![GPIO pins](https://github.com/stm32duino/Arduino_Core_STM32/assets/11053092/785aa817-6617-47f0-a37d-beb2a6237185)
![sch1](https://github.com/stm32duino/Arduino_Core_STM32/assets/11053092/eb35a517-d595-4168-8f4e-2b4819c7c19b)
![sch2](https://github.com/stm32duino/Arduino_Core_STM32/assets/11053092/22e5f1fa-45c3-4695-9f7a-21d34bfff32d)
![Board](https://github.com/stm32duino/Arduino_Core_STM32/assets/11053092/71247db8-f72e-4736-9f67-e8909cdc842c)
![code and display](https://github.com/stm32duino/Arduino_Core_STM32/assets/11053092/592ec0fc-1fb7-47e9-9def-28ea4b82a359)
